pkg/bdba/ctl_bdba.go

```diff
@@ -520,6 +523,8 @@ func (ctl *HelmValuesFromCobraFlags) AddHelmValueByCobraFlag(f *pflag.Flag) {
 			util.SetHelmValueInMap(ctl.args, []string{"frontend", "applicationLogging", "enabled"}, !ctl.flagTree.DisableFrontendLogging)
 		case "disable-worker-logging":
 			util.SetHelmValueInMap(ctl.args, []string{"worker", "applicationLogging", "enabled"}, !ctl.flagTree.DisableWorkerLogging)
+		case "log-retention":
+			util.SetHelmValueInMap(ctl.args, []string{"logRetention"}, ctl.flagTree.LogRetention)
```

Comment:
Just checking to make sure you want to set this as an `int`, not a `string`. (ctl.flagTree.LogRetention is from an Integer Flag)

See here: https://github.com/protecode-sc/helm-chart/pull/2/files#r447687576